### ISSUE TITLE
Share feature doesn't work although it says it does

### ISSUE THREAD

**Issue by leolivier** [wallabag/android-app#748]
When I'm in an application (let's say Firefox or chrome for the most reknown), and I'm using the share feature to remember a web page in wallabag, I click on the wallabag icon on the list of icons and I get a message saying 'added' but it's actually never added to wallabag.
I'm on a brand new Honor 8x but I had exactly the same issue on a OnePlus 5. 

Edit: I'm using my own wallabag installation on Raspbian stretch, the connection works fine, I can read my articles on my phone even though the sync is sometimes malfunctioning

**Comment by leolivier** [comment]
Hi @kai11
I don't think it's this kind of things as it used to work a while ago and it doesn't work anymore even with the smallest article...
And there's only this in the error log:

```pi@raspberrypi /var/log/apache2 $ tail wallabag-error.log
[Sun Dec 09 20:29:17.396140 2018] [proxy_fcgi:error] [pid 6274] [client 192.168.0.254:58872] AH01067: Failed to read FastCGI header
[Sun Dec 09 20:29:17.396358 2018] [proxy_fcgi:error] [pid 6274] (104)Connection reset by peer: [client 192.168.0.254:58872] AH01075: Error dispatching request to :
[Sun Dec 09 20:29:23.927438 2018] [proxy_fcgi:error] [pid 16932] [client 192.168.0.254:58878] AH01067: Failed to read FastCGI header
[Sun Dec 09 20:29:23.927603 2018] [proxy_fcgi:error] [pid 16932] (104)Connection reset by peer: [client 192.168.0.254:58878] AH01075: Error dispatching request to :
[Sun Dec 09 20:29:29.520348 2018] [proxy_fcgi:error] [pid 16926] [client 192.168.0.254:58880] AH01067: Failed to read FastCGI header
[Sun Dec 09 20:29:29.521333 2018] [proxy_fcgi:error] [pid 16926] (104)Connection reset by peer: [client 192.168.0.254:58880] AH01075: Error dispatching request to : 

```

**Comment by Strubbl** [comment]
The "AH01075: Error dispatching request to :" implies that this is an Apache misconfiguration.

**Comment by leolivier** [comment]
Ok, I'll have a look but I didn't change it as far as I remember.
What has been changed is the Apache version and maybe the fast cgi stuff linked to the version...

**Comment by Strubbl** [comment]
Sorry, i do not have enough information to help. Can you please provide more information? E.g. info about your wallabag installation.

What i am confused with:

>  In the app, no way to sync and no way to share with always the above obscure messages.

Is this issue also about syncing in general? Or only syncing the articles which have been shared to the app via e.g. Firefox?
When exactly does the syncing fail? The connection wizard of the app works without errors? Do you get any errors when syncing?

Do you have any redirects of the URL you have given to wallabag app? From your mobile, are you able to access $wallabagURL/api/version e.g. with Firefox?

**Comment by leolivier** [comment]
Yes, I realized yesterday that the same messages appear in the apache error log when I try to sync even the articles created from my pc. (And those ones, I can see them in the web interface)
The connection wizard shows no error, I reran it this morning. The version api also works and return"2.3.5" in my mobile's firefox.

**Comment by chrstn-hntschl** [comment]
Same problem with my installation:
wallabag 2.3.5 from the dockerhub container wallabag/wallabag:2.3.5
Adding articles via firefox extension works fine
I just reinstalled the android app and reconfigured. The app downloads all articles as expected, however, adding new articles is not possible (neither via android share nor via add article from app menu) - the articles don't show up in the app. I dont see any log message in the container except for this:
[11/Jan/2019:21:33:16 +0000] "POST /api/entries.json HTTP/2.0" 500 72 "-" "okhttp/3.8.1"

Appears whenever I refresh the app contents.

Any ideas?

**Comment by chrstn-hntschl** [comment]
Update: after realizing that the firefox extension does not work either - sorry for the confusion - I tried to add an article from the website and received the SQL error:
SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x98\x88.<...' for column 'content' at row 1

After adding the SYMFONY__ENV__DATABASE_CHARSET=utf8mb4 environment variable to the container, and restarting the container everything works fine now. Don't understand why I have to add this parameter all of a sudden - it used to work without in earlier versions of wallabag...

**Comment by Strubbl** [comment]
@chrstn-hntschl can you please open a PR or an issue in https://github.com/wallabag/docker ?

**Comment by chrstn-hntschl** [comment]
@Strubbl I found a related issue which I hijacked: https://github.com/wallabag/docker/issues/111

**Comment by di72nn** [comment]
Error notifications fixed with #794, also added ability to export local changes with #915.
